include/GafferScene/InteractiveRender.h

```diff
@@ -83,13 +84,20 @@ class GAFFERSCENE_API InteractiveRender : public Gaffer::Node
 		GafferScene::ScenePlug *outPlug();
 		const GafferScene::ScenePlug *outPlug() const;
 
+		Gaffer::ObjectPlug *messagesPlug();
+		const Gaffer::ObjectPlug *messagesPlug() const;
+
 		/// Specifies a context in which the InteractiveRender should operate.
 		/// The default is null, meaning that the context of the ancestor
 		/// ScriptNode will be used, or failing that, a default context.
 		void setContext( Gaffer::ContextPtr context );
 		Gaffer::Context *getContext();
 		const Gaffer::Context *getContext() const;
 
+		virtual void affects( const Gaffer::Plug *input, AffectedPlugsContainer &outputs ) const override;
+
+		IECore::MessageHandler *messageHandler() const;
```

Comment:
Perhaps it'll become obvious why during the rest of the review, but I wasn't expecting this to be exposed. I thought the `messagesPlug()` was the public interface?

Reply:
Ahh yeah, it was so we could have an integration test for the message handling overhead in `InteractiveRender` without needing a mock renderer, namely https://github.com/GafferHQ/gaffer/pull/3790/commits/9a7b69143e80c4e4ab925678bcc7322ea95f38d4

Reply:
As per our chat  https://github.com/GafferHQ/gaffer/pull/3790/commits/d6517e5ef41dae2643a920256283997d06f822d9 and https://github.com/GafferHQ/gaffer/pull/3790/commits/b2220b3d56bfc3103f0a121739127fcc7ec9de3b